packages/bootstrap/extra/dcos_internal_utils/bootstrap/cli.py

```diff
@@ -84,7 +84,3 @@ def parse_args():
         default='/opt/mesosphere/etc/master_count',
         help='File with number of master servers')
     return parser.parse_args()
-
-
-if __name__ == '__main__':
-    main()
```

Comment:
nit: should remove the shebang at the top as well, but needing to land this quickly as a _lot_ of builds are borked currently.